openhtf/output/web_gui/__init__.py

```diff
@@ -80,7 +80,6 @@
              default_value=[],
              description='List of manually declared stations.')
 
-
 Hostport = collections.namedtuple('Hostport', ['host', 'port'])
 
 
```

Comment:
For a little bit of context, I believe when we originally intended to take in stride occasional missed updates as a fact of life. That's why we _send_ complete state instead of deltas, even though the deltas are computed and applied based on the complete state we have and the complete state we receive.